src/main/resources/static/css/home-page-style.css

```diff
@@ -0,0 +1,77 @@
+body {
+    background-color: #42BFDD;
```

Comment:
I'd consider using css variables / custom properties to store colours for reuse

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties